app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -1920,22 +1920,22 @@ Redis connection timeout in milliseconds.
 
 #### config.session_redis_read_timeout
 
-Redis read timeout in milliseconds (not supported with Redis cluster).
+Redis read timeout in milliseconds for the Redis client.
```

Comment:
```suggestion
The read timeout in milliseconds for the Redis client.
```
Slight rephrase for two reasons:
1. "Redis read" sounds like a typo for "Redis reads". 
2. Don't need to say Redis twice in the sentence.